fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/socialmedia/ChirpListAdapterTest.java

```diff
@@ -27,10 +28,20 @@
 @HiltAndroidTest
 @RunWith(AndroidJUnit4.class)
 public class ChirpListAdapterTest {
+  private static final long CREATION_TIME = 1631280815;
+  private static final String LAO_NAME = "laoName";
+
+  private static final KeyPair SENDER_KEY_1 = generatePoPToken();
+  private static final KeyPair SENDER_KEY_2 = generatePoPToken();
+
+  private static final PublicKey SENDER_1 = SENDER_KEY_1.getPublicKey();
+  private static final PublicKey SENDER_2 = SENDER_KEY_2.getPublicKey();
+
+  private static final String LAO_ID = Lao.generateLaoId(SENDER_1, CREATION_TIME, LAO_NAME);
 
   private final Intent intent =
       new Intent(ApplicationProvider.getApplicationContext(), SocialMediaActivity.class)
-          .putExtra("OPENED_FROM", "");
+          .putExtra(Constants.LAO_ID_EXTRA, "");
```

Comment:
Just for the sake of consistency, could you set the LAO_ID ?
```suggestion
          .putExtra(Constants.LAO_ID_EXTRA, LAO_ID);
```